fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentMapper.java

```diff
@@ -67,9 +87,96 @@ public FluentClient map(CodeModel codeModel, Client client) {
             fluentClient.getExamples().addAll(examples);
         }
 
+        // live tests
+        fluentClient.getLiveTests().addAll(client.getLiveTests().stream().map(liveTests -> mapLiveTests(liveTests, fluentClient)).collect(Collectors.toList()));
+
         return fluentClient;
     }
 
+    private FluentLiveTests mapLiveTests(LiveTests liveTests, FluentClient fluentClient) {
+
+        FluentLiveTests result = new FluentLiveTests();
+        result.setClassName(liveTests.getFilename() + "Tests");
+
+        parseLiveTests: for (LiveTestCase liveTestCase : liveTests.getTestCases()) {
+            FluentLiveTestCase testCase = new FluentLiveTestCase();
+            testCase.setMethodName(liveTestCase.getName());
+            for (LiveTestStep liveTestStep : liveTestCase.getTestSteps()) {
+                if (liveTestStep instanceof ExampleLiveTestStep) {
+                    FluentExampleLiveTestStep fluentStep = new FluentExampleLiveTestStep();
+                    ExampleLiveTestStep exampleStep = (ExampleLiveTestStep) liveTestStep;
+                    String operationId = exampleStep.getOperationId();
+                    String[] oprs = operationId.split("_");
```

Comment:
What if it does not have operationGroup?

Reply:
Thanks for the hint, done.